integration/smartbft/smartbft_block_deliverer_test.go

```diff
@@ -95,16 +95,14 @@ var _ = Describe("Smart BFT Block Deliverer", func() {
 
 		/* Create a network from the networkConfig */
 		network = nwo.New(networkConfig, testDir, client, StartPort(), components)
-		network.Profiles[0].SmartBFT = &nwo.SmartBFT{
-			LeaderHeartbeatTimeout: 10,
-			LeaderHeartbeatCount:   10,
-		}
 		network.GenerateConfigTree()
 		network.Bootstrap()
 
 		for _, orderer := range network.Orderers {
-			runner := network.OrdererRunner(orderer)
-			runner.Command.Env = append(runner.Command.Env, "FABRIC_LOGGING_SPEC=debug")
+			runner := network.OrdererRunner(orderer,
+				"FABRIC_LOGGING_SPEC=debug",
+				"ORDERER_GENERAL_BACKOFF_MAXDELAY=20s",
```

Comment:
why do we need the max delay?

Reply:
These are my recommendations for setting up grpc so that some individual orderer can't leave the consensus due to lack of heartbeat from the leader.

The author added the settings, most likely by copying them:
```
                network.Profiles[0].SmartBFT = &nwo.SmartBFT{
			LeaderHeartbeatTimeout: 10,
			LeaderHeartbeatCount:   10,
		}
```

These settings are not needed here. I've tweaked it as it should be.